source/documentation/testing-process.html.md.erb

```diff
@@ -1,13 +1,33 @@
 ---
 title: Testing process
-weight: 15
+weight: 30
 ---
 
 # Testing process
 
-This is a prototype testing guide used for research. The testing process it describes is still in development.
-
 You must test the compatibility of your software with [EMCS API](https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/excise-movement-control-system-api/1.0).
 
-You will be able to connect to the [API in External Test]().
+After [obtaining a bearer token](obtain-a-bearer-token.html), use postman or curl to call the API at: <br></br> `https://test-api.service.hmrc.gov.uk/customs/excise/movements`
+
+The following values in angled brackets (<>) must be replaced:
+
+- `<your-access-token>` - your [bearer token](obtain-a-bearer-token.html). Make sure to replace the angled brackets (<>) as well.
+- `<urn:IE815>...</urn:IE815>` - keep the angled brackets (<>), and replace with the relevant IE number and [XML for the message you are testing](https://github.com/hmrc/excise-movement-control-system-api/tree/main/app/xsd).
```

Comment:
If we do use [] I guess we don't need to worry about confusing them between XML and replacements

Reply:
Replaced with Create a Test User, corrected use of []